src/screens/Settings.js

```diff
@@ -95,6 +96,35 @@ class Settings extends React.Component {
     }
   }
 
+  /**
+   * When user clicks Export Registered Tokens button, then we save all config strings in a txt file
+   */
+  exportTokens = () => {
+    // The file text will be the configuration strings of each registered token, one each line
+    //
+    // First we get all token configs from registered tokens array,
+    // remove the HTR token with filter, then map to each configuration string
+    const configurationStrings = this.props.registeredTokens.filter((token) => {
+      return token.uid !== hathorLib.constants.HATHOR_TOKEN_CONFIG.uid;
+    }).map((token) => {
+      return hathorLib.tokens.getConfigurationString(token.uid, token.name, token.symbol);
+    });
+
+    // The text will be all the configuration strings, one for each line
+    const text = configurationStrings.join('\n');
+
+    // Create the hidden a element to trigger the download
+    const element = document.createElement('a');
+    const file = new Blob([text], {
+      type: 'text/plain'
+    });
+    element.href = URL.createObjectURL(file);
+    element.download = 'Hathor Wallet - Tokens.txt';
+    document.body.appendChild(element);
+    element.click();
+    element.remove();
```

Comment:
I know this is required when downloading a blob from a browser, but why can't we use the `fs` module?